packages/dx-react-scheduler/src/plugins/week-view.tsx

```diff
@@ -59,58 +100,66 @@ export class WeekView extends React.PureComponent {
       intervalCount,
     } = props;
 
-    const viewCellsDataComputed = ({
+    const viewCellsDataBaseComputed = ({
       currentDate,
     }) => viewCellsDataCore(
       currentDate, firstDayOfWeek,
-      intervalCount * DAYS_IN_WEEK, excludedDays,
-      startDayHour, endDayHour, cellDuration,
+      intervalCount! * DAYS_IN_WEEK, excludedDays!,
+      startDayHour!, endDayHour!, cellDuration!,
       Date.now(),
     );
 
     const timeTableElementComputed = () => this.timeTable;
     const layoutElementComputed = () => this.layout;
     const layoutHeaderElementComputed = () => this.layoutHeader;
 
-    this.currentViewComputed = ({ currentView }) => (
+    this.sidebarPlaceholder = () => <TemplatePlaceholder name="sidebar" />;
```

Comment:
These placeholders and similar methods can be moved outside constructor to reduce it's size so 

```
dayScalePlaceholder: any;
constructor() {
  ...
  this.dayScalePlaceholder = () => <TemplatePlaceholder name="navbar" />;
}
```

can be replaced with
```
constructor() {
  ...
}
dayScalePlaceholder = () => <TemplatePlaceholder name="navbar" />;
```